src/components/timecard/select-timecard-period-type/SelectTimecardPeriodType.jsx

```diff
@@ -24,6 +16,23 @@ const SelectTimecardPeriodType = () => {
     shouldFocusError: false,
   });
 
+  const [timePeriods, setTimePeriods] = useState([]);
+  const getTimePeriodTypeData = async () => {
+    try {
+      const response = await getTimePeriodTypes({
+        tenantId: '00000000-0000-0000-0000-000000000000',
+      });
```

Comment:
The `saveTimeEntry` function in `EditShiftHours` passes the tenant ID in a `URLSearchParams` object, maybe we should keep them consistent?

Reply:
Good point. Incorporated